python/mxnet/image.py

```diff
@@ -37,8 +37,8 @@ def imdecode(buf, **kwargs):
         1 for three channel color output. 0 for grayscale output.
     to_rgb : int, optional, default=1
         1 for RGB formatted output (MXNet default). 0 for BGR formatted output (OpenCV default).
-    out : NDArray, optional
-        Output buffer. Use `None` for automatic allocation.
+
+    # removed out parameter as it is not used
```

Comment:
Can you please point out where it is used? I am not able to find the relevant code in image_io.cc